src/esp/metadata/attributes/LightLayoutAttributes.cpp

```diff
@@ -15,8 +17,8 @@ LightInstanceAttributes::LightInstanceAttributes(const std::string& handle)
   setIntensity(1.0);
   setType("point");
   // ignored for all but spot lights
-  setInnerConeAngle(0.75);
-  setOuterConeAngle(1.5);
+  setInnerConeAngle(0.75_radf);
+  setOuterConeAngle(1.5_radf);
```

Comment:
Hmm, maybe use the same defaults as glTF has? So `0.0_degf` and `90.0_degf`? (glTF uses half-angles, so there it's 45°). 

(And I have to fix [LightData](https://doc.magnum.graphics/magnum/classMagnum_1_1Trade_1_1LightData.html), because there I have the defaults wrong, heh.)